main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"Fibbonaci/Data"
```

Comment:
Package names should be lower case.

https://github.com/uber-go/guide/blob/master/style.md#package-names